<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Duplicate Tracks</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>21</x>
      <y>10</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="directoryPath" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2" >
      <widget class="QPushButton" name="directoryBrowse" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="findDuplicates" >
       <property name="text" >
        <string>Find Duplicates</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5" >
      <widget class="QTableView" name="tableView" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="showGrid" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>332</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="fileCount" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLabel" name="fileCountEnd" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>files checked</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QLabel" name="duplicatesFoundCount" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4" >
      <widget class="QLabel" name="duplicatesFoundEnd" >
       <property name="text" >
        <string>duplicates found</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
